app/models.py

```diff
@@ -12,6 +12,10 @@ class Contract(models.Model):
     signed_date = models.DateField()
     event_id = models.CharField(max_length=80, null=True, blank=True)
     user_id = models.CharField(max_length=80, null=True, blank=True)
+    description = models.TextField()
```

Comment:
The text field in MySQL is slow and consumes much space. It's hard not recommend, Why is it not possible to use varchar in this case?

Reply:
We're using sqlite, does varchar apply ?

Reply:
I don't know so mucho about SQLite, could you make a quick review?

Reply:
Sqlite documentation says "SQLite does not impose any length restrictions", because everything is stored as strings. 
Also, we discovered Data types in SQLite are only there for sql compatibility.

https://www.sqlite.org/datatype3.html#affinity_name_examples